component/core/src/main/java/org/exoplatform/social/core/jpa/storage/dao/jpa/MetadataItemDAO.java

```diff
@@ -54,6 +54,8 @@ public class MetadataItemDAO extends GenericDAOJPAImpl<MetadataItemEntity, Long>
   private static final String OBJECT_ID            = "objectId";
 
   private static final String SPACE_ID             = "spaceId";
+  
+  private static final String SPACES_ID             = "spacesId";
```

Comment:
plural for last word and not for the middle word (it's a list of IDs of type space and not a lsit of spaces for an id)

```suggestion
  
  private static final String SPACE_IDS             = "spaceIds";
```